webview_panels/src/modules/documentationEditor/styles.module.scss

```diff
@@ -2,6 +2,16 @@
 
 
 .documentation-wrapper {
+  position: absolute;
```

Comment:
You will not need position absolute for this change. This element is already taking 100vw and vh

Reply:
`position: absolute` is required. Not doing that breaks the layout.
<img width="1654" alt="Screenshot 2024-07-26 at 12 19 51 PM" src="https://github.com/user-attachments/assets/8959626b-7e86-42fe-9b43-9240cf7006b9">